src/main/java/org/swisspush/redisques/action/AbstractQueueAction.java

```diff
@@ -53,12 +53,8 @@ public AbstractQueueAction(Vertx vertx, LuaScriptManager luaScriptManager, Redis
         this.log = log;
     }
 
-    public JsonObject createOkReply() {
-        return new JsonObject().put(STATUS, OK);
-    }
-
-    public JsonObject createErrorReply() {
-        return new JsonObject().put(STATUS, ERROR);
+    protected Handler<Throwable> replyErrorMessageHandler(Message<JsonObject> event) {
+        return throwable -> event.reply(new JsonObject().put(STATUS, ERROR));
```

Comment:
This was made by intention, since we reply most of the messages like this. Example in _GetLockHandler_:
```java
@Override
public void handle(AsyncResult<Response> reply) {
	if (reply.succeeded()) {
		if (reply.result() != null) {
			event.reply(new JsonObject().put(STATUS, OK).put(VALUE, reply.result().toString()));
		} else {
			event.reply(new JsonObject().put(STATUS, NO_SUCH_LOCK));
		}
	} else {
		event.reply(new JsonObject().put(STATUS, ERROR));
	}
}
```